src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/publicApi/PublicLicenceServiceTest.kt

```diff
@@ -45,7 +48,7 @@ class PublicLicenceServiceTest {
   private val additionalConditionRepository = mock<AdditionalConditionRepository>()
   private val additionalConditionUploadDetailRepository = mock<AdditionalConditionUploadDetailRepository>()
   private val licenceService = mock<LicenceService>()
-  private val modelLicenceMock = mock<uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.Licence>()
+  private val modelLicenceMock = mock<ModelLicence>()
```

Comment:
this can be removed as its not referenced anywhere

Reply:
Done